addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -516,7 +521,7 @@ msgstr ""
 
 #: /xml/VideoOSD.xml
 msgctxt "#31112"
-msgid "Toggle audio stream"
+msgid "Audio streams"
```

Comment:
I consider this change risky as you never know whether other skins also use this string for whatever. With this change you might "break" them. I would introduce a new string instead of changing the existing string.

Reply:
with each new version of kodi skinner they have to adjust their thirdy party skins,
so i dont see real reason to keep obsolete texts that are no longer used on Kodi...
there are already other unused strings on files that just cause a mess by occupying id's

Reply:
Have to admit I ran out of string space in 31000 block on aeon mq5 mods, so always am looking to the Kodi strings to "borrow" from.  Maybe not so much for something like "toggle audio stream" that is kind of use specific,  but "Subtitle" -> "Subtitle stream" could be considered to keep (just checked and I don't use it).  Note this also potentially affects any skinned addon windows.

Reply:
Then you need to document this breaking changes for the skinners in the forum. We have a thread for that there.

Reply:
yes i will add to https://forum.kodi.tv/showthread.php?tid=377306